balloon-communication/CMakeLists.txt

```diff
@@ -33,7 +35,9 @@ add_executable(wifibroadcast
 add_executable(udp_test_receiver
         udp/test/udp_test_receiver.c
         udp/test/udp_test_util.c
-        udp/udp_client.c udp/rx_udp_util.c)
+        udp/udp_client.c
+        #udp/rx_udp_util.c
```

Comment:
Good catch, `rx_udp_util.c` should definitely not be included in the test executable.